index.js

```diff
@@ -1,3 +1,17 @@
+/**
+ * Compile a string that contains wildcard `*` to a matcher function
+ *
+ * @param {string} wildcard The string that needed to be compiled
+ * @returns {function} The matcher function
+ */
+var compileWildcard = function (wildcard) {
+  var regexpstring = '^' + wildcard.replace(/[*]/g, '[a-zA-Z0-9]+')
```

Comment:
`wildcard` theoretically can contain other RegExp control symbols. You need to escape then.

Maybe allowing passing RegExp to `paths` is a better option? I know that `*` is very readable. But it is not flexible and require more code. It is OK for big thing, which you use in every file. But maybe for one config we can allow less readable RegExp?

Reply:
Thanks! I've made your suggested changes.